MovieDemo/MovieDemo/Views/MemosSheet.swift

```diff
@@ -0,0 +1,56 @@
+//
+//  MemosSheet.swift
+//  MovieDemo
+//
+//  Created by Anton Kölbl on 16.01.24.
+//
+
+import SwiftUI
+
+struct MemosSheet: View {
+  
+  @Binding var showSheet: Bool
+  
+  @EnvironmentObject var store: Store
+  
+  var body: some View {
+    VStack {
+      if store.memos.isEmpty {
+        ContentUnavailableView(
+          "No memos yet",
+          systemImage: "bookmark",
+          description: Text("You can create memos on the detail page for any movie.")
+        )
+      } else {
+        VStack(alignment: .center, spacing: 8) {
+          Text("All memos")
+            .font(.title)
+            .padding(20)
+          ScrollView {
+            ForEach(store.memoKeys, id: \.self) { key in
```

Comment:
consider some kind of divider line or else using a list here. When I added many memos, the page became confusing.